src/workerd/server/workerd.capnp

```diff
@@ -267,7 +267,7 @@ struct Worker {
       # A Python module. All bundles containing this value type are converted into a JS/WASM Worker
       # Bundle prior to execution.
 
-      pythonRequirement @9 :Text;
+      pythonRequirement @9 :Void;
```

Comment:
In general, you cannot change the type of an existing field in a Cap'n Proto schema without potentially breaking binary compatibility -- even for messages that do not use the specific field, since the sequence of field affects the layout of the struct. The types of changes that are safe are documented here:

https://capnproto.org/language.html#evolving-your-protocol

We do at least sort of care about binary compatibility for workerd config, as we allow people to compile a config into binary format and then load that in production. Whether or not anyone actually does this, I'm honestly not sure, but it is a thing we support via the CLI.

With all that said, it _just happens_ that this particular change will not break binary compatibility. This is essentially by coincidence, due to details of the encoding algorithm and the particular sequence of fields defined before this one. But in general this is not something you should try to guess about. You can always ask me to check.